src/rdd/rdd.rs

```diff
@@ -838,6 +859,65 @@ pub trait Rdd: RddBase + 'static {
         self.intersection_with_num_partitions(other, self.number_of_splits())
     }
 
+    fn subtract<T>(&self, other: Arc<T>) -> SerArc<dyn Rdd<Item = Self::Item>>
```

Comment:
If possible, please add documentation for this method. Similar to the one in Spark will do.

Reply:
have added documentation for the function and an example in examples/subtract.rs